api/tests/integration/application/organization-invitations/organization-invitation-controller_test.js

```diff
@@ -90,4 +93,69 @@ describe('Integration | Application | Organization-invitations | organization-in
     });
   });
 
+  describe('#sendSCOInvitation', () => {
+
+    const uai = '1234567A';
```

Comment:
si tu veux rester cohérent avec les autres modifs
```suggestion
    const UAI = '1234567A';
```